### ISSUE TITLE
The __init__.py in package_assembly

### ISSUE THREAD

**Issue by tsvenson** [dabeaz/modulepackage#1]
Hi David,

Done with Python 3.4.3 on Ubuntu 14.04 LTS

I'm following your PyCon 2015 training session as linked to on http://dabeaz.com/modulepackage/ and got a bit stuck on this error message PyCharm 5 gives when hovering over "foo." and "bar." in the `__init__.py` in the "package_assembly" directory:
- "? lazy_assembly.spam.bar? (multiple choices...) Alt+Enter"

Then if I press "Alt+Enter" (when caret is on either "foo." or "bar." I get this dropdown menu:

![Import from ".foo|.bar"](https://cloud.githubusercontent.com/assets/444455/12074858/188b9274-b16c-11e5-9e1f-9b3ff1654a40.png)

Those packages listed have two things in common, they all are in the same directory and they all contain a "spam" sub-folder with their own "`__init__.py`" file.

Basically, all file "spam" folders are on a parallel directory level to this one and Python/PyCharm doesn't know which one to pic.

Like you I don't get any errors/tracebacks when running "import spam", but which of the five places is it really imported from?
